data.md

```diff
@@ -286,6 +286,19 @@ The function interprets the range of bytes as little-endian.
       requires START <Int  END
 ```
 
+External Values
+---------------
+
+An `external value` is the runtime representation of an entity that can be `imported` or `exported`. It is an `address` denoting either a `function instance`, `table instance`, `memory instance`, or `global instances` in the shared store.
+
+```k
+    syntax Externval ::= "func"   TextFormatIdx
+                       | "table"  TextFormatIdx
+                       | "mem"    TextFormatIdx
+                       | "global" TextFormatIdx
+ // -------------------------------------------
```

Comment:
From what I can tell, only `"func" TextFormatIdx` is handled in this PR. Can we remove the others until they are handled?

Reply:
fixed